clientapi/routing/routing.go

```diff
@@ -538,7 +540,7 @@ func Setup(
 	).Methods(http.MethodGet, http.MethodOptions)
 
 	v3mux.Handle("/account/password",
-		httputil.MakeAuthAPI("password", userAPI, func(req *http.Request, device *userapi.Device) util.JSONResponse {
+		httputil.MakeContitionalAuthAPI("password", userAPI, func(req *http.Request, device *userapi.Device) util.JSONResponse {
```

Comment:
MakeConditionalAuthAPI

Reply:
@Danieloni1 comment refers to outdated code, I have changed it